packages/subgraph/src/controller.ts

```diff
@@ -85,13 +90,21 @@ export function handleBurnShort(event: BurnShort): void {
 
   let timestamp = event.block.timestamp;
   let transactionHash = event.transaction.hash.toHex();
+
+  //check if users manually burn or using shorthelper to close position
+  let actionType: string;
+  if (event.transaction.to === SHORT_HELPER_ADDR) {
```

Comment:
event.transaction .from

Reply:
@KMKoushik I thought tx.from is user’s address and tx.to is the contract address🤔

Reply:
Sorry I made a mistake. There will be 2 addresses for a transaction like shortHelper. i) Address initiated the transaction(msg.from) ii) Sender of this particular call (msg.sender). For example consider this tx: https://etherscan.io/tx/0x54be2cc4ed0dc633be21b2a702cf10f19b15905359eaa0498e0b40caa1693fb3

Which is initiated by 0x9b6ab9b2acbdb25a56971daedd7b3e8c4c1364ad. But if you check events from logs tab sender address will shortHelper's address 

<img width="1086" alt="Screenshot 2022-02-03 at 9 33 19 AM" src="https://user-images.githubusercontent.com/24666922/152279322-9f04dd71-4570-4e4f-97e7-e0bbc0a88703.png">

So instead of using `event.transaction.to` use event.params.sender (which uses msg.sender)

<img width="825" alt="Screenshot 2022-02-03 at 9 35 55 AM" src="https://user-images.githubusercontent.com/24666922/152279498-2a299491-cbbb-478f-979a-defafaa9cc17.png">